static/templates/compose.hbs

```diff
@@ -62,9 +62,6 @@
                     <div id="stream-message">
                         <div class="stream-selection-header-colorblock message_header_stream left_part"></div>
                         <div class="right_part">
-                            <span id="compose-lock-icon">
```

Comment:
removed this hard-coded span with lock icon - doesn't make sense for it to be here